packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-configuration-item.vue

```diff
@@ -77,30 +73,42 @@ const onDeleteConfiguration = () => {
 </script>
 
 <style scoped>
-.config-card {
+div {
 	background: var(--surface-0);
 	border-left: 4px solid var(--surface-300);
+	padding: var(--gap-1) var(--gap-1) var(--gap-3) var(--gap-3);
+
 	&:hover {
-		cursor: pointer;
 		background-color: var(--gray-50);
+		cursor: pointer;
+	}
+
+	&.selected {
+		border-left-color: var(--primary-color);
 	}
-	&.card-selected {
-		border-left: 4px solid var(--primary-color);
+
+	&,
```

Comment:
what is `&,`

Reply:
It translate to `div, div.selected`.
It is a way to write the `transition` property once with two characters.